_assets/scss/elements/_typography.scss

```diff
@@ -88,6 +99,7 @@
 .h5--med { @include h5--med; }
 .h5--med { @include h5--med; }
 
+// $fix – box works on Chrome but not on safari
```

Comment:
Thoughts on how to fix? Remove the focusBox stuff? It was a design requirement called out here: https://rationale.dapulse.com/boards/47332893/pulses/50953055

If it's causing too many problems we can go back to the browser default.